x-pack/libbeat/magefile.go

```diff
@@ -39,8 +39,3 @@ func Fields() error {
 func GoTestUnit(ctx context.Context) error {
 	return devtools.GoTest(ctx, devtools.DefaultGoTestUnitArgs())
 }
-
-// Config generates example and reference configuration for libbeat.
-func Config() error {
-	return devtools.Config(devtools.ShortConfigType|devtools.ReferenceConfigType, devtools.ConfigFileParams{}, ".")
-}
```

Comment:
@andrewkroh this target was creating `libbeat.reference.yml` and `libbeat.yml`, that were not being commited. Not sure if these files were expected. Wdyt?

Reply:
I guess we don't need those since there is no real Beat here. And the passing tests would indicate that they are not needed for the "mockbeat" system tests.